.github/workflows/ik-ci.yml

```diff
@@ -0,0 +1,45 @@
+name: InstrumentKit CI
+
+on: [push]
```

Comment:
Should probably extend this to also run on pull-requests. In addition, could specify to only run on master, i.e., on a push to master or a PR request to master, something like this:

``` yaml
on:  
  push:
  pull_request:
    branches:
      - master
```

Reply:
Yeah, mostly just been testing so far to see what works and doesn't